subsys/bluetooth/mesh/Kconfig

```diff
@@ -316,6 +323,28 @@ config BT_MESH_LOOPBACK_BUFS
 	  The number of buffers allocated for the network loopback mechanism.
 	  Loopback is used when the device sends messages to itself.
 
+config BT_MESH_TX_SEG_RETRANS_CNT
```

Comment:
Sortening "COUNT" to "CNT" doesn't really do much here. The variable would anyway not be the longest one that you're introducing, so just spell out COUNT.